onnxruntime/core/providers/cpu/math/top_k.cc

```diff
@@ -170,16 +234,47 @@ template <>
 Status TopK<10, float>::Compute(OpKernelContext* p_op_kernel_context) const {
   const auto* X = p_op_kernel_context->Input<Tensor>(0);
   const auto* Y = p_op_kernel_context->Input<Tensor>(1);
-  if (X == nullptr || Y == nullptr) return Status(common::ONNXRUNTIME, common::FAIL,
+  if (X == nullptr || Y == nullptr) return ORT_MAKE_STATUS(ONNXRUNTIME, FAIL,
                                                   "input count mismatch, expected 2 inputs - "
                                                   "the tensor to be processed and a tensor containing k value");
   const vector<int64_t>& y_shape = Y->Shape().GetDims();
-  if (y_shape.size() != 1 || y_shape[0] != 1) return Status(common::ONNXRUNTIME, common::FAIL, "k tensor should be a 1D tensor of size 1");
+  if (y_shape.size() != 1 || y_shape[0] != 1) return ORT_MAKE_STATUS(ONNXRUNTIME, FAIL, "k tensor should be a 1D tensor of size 1");
   auto parsed_input_k = Y->template Data<int64_t>()[0];
-  if (parsed_input_k < 0) return Status(common::ONNXRUNTIME, common::FAIL, "value of k must not be negative");
+  if (parsed_input_k < 0) return ORT_MAKE_STATUS(ONNXRUNTIME, FAIL, "value of k must not be negative");
   return TopKImpl(p_op_kernel_context, X, axis_, gsl::narrow_cast<unsigned>(parsed_input_k));
 }
 
+// Opset ver - 11
+template <>
+TopK<11, float>::TopK(const OpKernelInfo& op_kernel_info) : OpKernel(op_kernel_info) {
+  int64_t axis_temp;
+  ORT_ENFORCE(op_kernel_info.GetAttr<int64_t>("axis", &axis_temp).IsOK());
+  axis_ = gsl::narrow_cast<int>(axis_temp);
+
+  int64_t largest_temp;
+  ORT_ENFORCE(op_kernel_info.GetAttr<int64_t>("largest", &largest_temp).IsOK());
+  largest_ = largest_temp == 1 ? true : false;
```

Comment:
GetAttrOrDefault

Reply:
I believe we don't need to use this. The ONNX schema for the op contains the default values for the attributes. Its only in cases that the schema doesn't hold the default value should we use this method to specify our understanding of the default value...